src/renderers/webxr/WebXRManager.js

```diff
@@ -378,7 +373,38 @@ class WebXRManager extends EventDispatcher {
 			for ( let i = 0; i < event.added.length; i ++ ) {
 
 				const inputSource = event.added[ i ];
-				const controller = inputSourcesMap.get( inputSource );
+
+				let controllerIndex = controllerInputSources.indexOf( inputSource );
+
+				if ( controllerIndex < 0 ) {
```

Comment:
I think this is what we're checking, right?

```js
if ( controllerIndex === -1 ) {
```

Reply:
Yes